src/app/guards/cancel-edit-tech/cancel-edit-tech.guard.ts

```diff
@@ -8,9 +8,14 @@ import { TechRecordComponent } from 'src/app/features/tech-record/tech-record.co
 @Injectable({
   providedIn: 'root'
 })
-export class CancelEditTechGuard implements CanDeactivate<TechRecordComponent> {
+export class CancelEditTechGuard implements CanDeactivate<TechRecordComponent>, CanActivate {
   constructor(private store: Store<State>) {}
 
+  canActivate(): boolean {
+    this.store.dispatch(updateEditingTechRecordCancel());
+    return true;
+  }
```

Comment:
From what I understand, this refreshes the `editingTechRecord`, right?

Reply:
Yeah so basically it deletes the editingTechRecord, then the component is forced to properly update the editingTechRecord (I think it's actually on the resolver on the route)